### ISSUE TITLE
Duplicated label `project_id` for exported Stackdriver Metrics

### ISSUE THREAD

**Issue by svenmueller** [google/slo-generator#99]
We recently switched to version `1.3.2` (we were running version `1.1.1` before). Since then we noticed an issue with exporting the custom Stackdriver metrics to Prometheus (using Prometheus expoter `stackdriver-exporter`) due to duplicated label `project_id`.

Example (note: prometheus Stackdriver exporter adds a prefix to the exported metric):
```
stackdriver_global_custom_googleapis_com_sli_measurement{alerting_burn_rate_threshold="9.0",error_budget_policy_step_name="1 hour",feature_name="products-update",project_id="test",project_id="test",service_name="projects",slo_name="availability",unit="",window="3600"}
```

There was a change recently here to update Stackdriver metric labels: https://github.com/google/slo-generator/commit/268058ada37a61f1d797868017d1ee33d2e7c37f#diff-0fabb485d257541a1302d5eec2d473307643ce28de6f1ffec68e7c50612b66e0R60

Note: Before the version upgrade exporting custom Stackdriver metrics like `error_budget_burn_rate` using Prometheus `stackdriver-exporter` was working as expected.

**Comment by ocervell** [comment]
Opened #100 to fix this

**Comment by ocervell** [comment]
Fixed by #100
